### DISCUSSION TITLE
Welcome to finplot Discussions!

### DISCUSSION THREAD

**Discussion by highfestiva** [highfestiva/finplot#72] [General]
Fire away! If you think you've got a bug, create a tiny reproduction and go to issues instead.

**Comment by engineertree5** [comment]
NO BUG AT ALL!!! I"M JUST EXCITED BEYOND BELIEF!!! JUST WANT TO SHARE MY EXCITMENT! THANK YOU HIGHFESTIVA FOR DOING THE WORK TO MAKE FINPLOT. 5 STARS! LOVE IT!

**Reply by highfestiva** [reply]
THANKS MAN!!! :D

**Reply by karunkrishna** [reply]
I SECOND THIS EXCITMENT!!!! THIS IS SUCH A GREAT PACKAGE!!!

**Reply by highfestiva** [reply]
YOU GUYS PUT A SMILE ON MY FACE!!! OPEN SOURCE FTW!!!

**Comment by AlexStox** [comment]
I don't understand! Your Finplot is a super solution: light, easy to use, beautiful! I would say brilliant. Why don't you have many stars and followers?!

**Reply by highfestiva** [reply]
It's growing organically without the help of some major lib (like matplot -> mplfinance). But I'm very satisfied with the [exponential growth so far](https://starchart.cc/highfestiva/finplot.svg).
